modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -3128,6 +3132,19 @@ public void testDeleteObjectEntry() throws Exception {
 
 		_objectEntryLocalService.deleteObjectEntry(objectEntry1);
 
+		List<SystemEvent> systemEvents =
```

Comment:
@DaniSzimko @ccorreagg could you place this new code right after 
```
_objectEntryLocalService.deleteObjectEntry(
			objectEntry1.getObjectEntryId());
```
(row 3124) ? Thx